docs/en/userguide/index.md

```diff
@@ -0,0 +1,47 @@
+title: Installed Modules Report
+summary: Assists with the maintenance of your SilverStripe application
+
+# Installed Modules Report
+## Usage 
+
+SilverStripe websites and the CMS can be highly customised using SilverStripe modules. The 'Installed modules' report 
+provides an overview of the modules installed for your SilverStripe website. 
+
+In the admin section of your SilverStripe website navigate to the Reports section. Open up the 'Installed modules' 
+report to view the available information. 
+
+Please note that you are required to have admin access to view this information.
+
+## Check for updates
+
+Click on the 'Check for Updates' button to refresh the list with the latest available information. Updating this list 
+may take 2-3 minutes. You can continue to use the CMS while the update is run.
```

Comment:
I think it's important to explain what actually happens here. Instead of "may take 2-3 minutes" we could say something like "may take some time, since it's waiting for the system to run a scheduled job in the background" or something like that